clusters/l3-sqnc/monitoring/values.yaml

```diff
@@ -0,0 +1,79 @@
+alertmanager:
+  enabled: true
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  config:
+    receivers:
+      - name: "null"
+      # - 
+      # - name: slack
+      #   webhook_configs:
+      #     - send_resolved: true
+      #       url: <secret>
+prometheus:
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  prometheusSpec:
+    retention: 24h
+    resources:
+      requests:
+        cpu: 200m
+        memory: 200Mi
+    serviceMonitorSelectorNilUsesHelmValues: false
+    serviceMonitorNamespaceSelector: { }
+    podMonitorSelectorNilUsesHelmValues: false
+    podMonitorNamespaceSelector: { }
+    podMonitorSelector:
+      matchLabels:
+        app.kubernetes.io/component: monitoring
+grafana:
+  defaultDashboardsEnabled: false
+  serviceMonitor:
+    enabled: true
+    namespace: monitoring
+  persistence:
+    enabled: true
+  adminPassword: grafana
```

Comment:
```suggestion
```
We should set a better default using the valuesFrom `secret` in the `release.yaml` instead.